eager-feedback-app.js

```diff
@@ -6,36 +6,35 @@
   var options, isPreview, locationsCSSMap, style, button, el, getRadiosValue, form, show, hide, toggle;
 
   options = INSTALL_OPTIONS;
+  isPreview = INSTALL_ID === "preview";
```

Comment:
This should be the preferred way of detecting this from now on, as it will also work with local apps (and it's shorter, and doesn't require knowing the app's id).